test-no-uncommitted-doc-changes.in

```diff
@@ -20,7 +20,7 @@ if [ "$PLONE_VERSION" = "5.0.x" ]; then
     exit 0
 fi
 
-changes=$(git diff --exit-code $DUMPS_DIR)
+changes=$(git diff --ignore-space-at-eol --exit-code $DUMPS_DIR)
```

Comment:
Yep, my editor removes whitespace at EOL. I think I'm not the only one, but don't down on that ;)